js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -2171,7 +2171,12 @@ var EditingController = modules.ViewController.inherit((function() {
 
         _afterSaveEditData: function() { },
 
-        _beforeCancelEditData: function() { },
+        _beforeCancelEditData: function() {
+            if(CELL_BASED_MODES.indexOf(getEditMode(this)) !== -1 && this.component.cellValue) {
```

Comment:
Why we need check "this.component.cellValue"? For tests?

Reply:
Yes, for dataController tests.

Reply:
In this case I recommend you add "noop" "cellValue" function in the related mock test object